src/docs/development/accessibility-and-localization/internationalization.md

```diff
@@ -275,8 +277,38 @@ that produces an instance of `GlobalMaterialLocalizations`.
 As of February 2020, the global localization classes support
 [77 languages][].
```

Comment:
I _believe_ that this is still accurate... I went into the translation console and hand-counted the list because I'm not sure how else to do it. @HansMuller where do we pull this count from? Does the count also distinguish between regional locale support like Chinese traditional and Chinese simplified?

Reply:
Great question! @HansMuller?

Reply:
This is a count of the locales supported in that bulleted list seen in [GlobalMaterialLocalizations](api.flutter.dev/flutter/flutter_localizations/GlobalMaterialLocalizations-class.html). Turns out we now support 78 languages!